Backend/AlgorithmsManager.py

```diff
@@ -557,6 +562,21 @@ def get_args_as_html_str(required, optional):
         return htmlstr
 
 
+    @staticmethod
```

Comment:
just out of curiosity- why is this static?

Reply:
It is now switched to an instance function. This was just a temporary implementation